lte/gateway/deploy/roles/dev_common/tasks/main.yml

```diff
@@ -158,6 +158,12 @@
   shell: gem install fpm
   when: preburn
```

Comment:
Nit: Usually, we just set the step to "full_provision" with a TODO rather than duplicating it.

Reply:
Done